sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -344,6 +344,57 @@ def equals(self, other):
               requires_partition_by=partitionings.Singleton(),
               preserves_partition_by=partitionings.Singleton()))
 
+  @frame_base.args_to_kwargs(pd.DataFrame)
```

Comment:
It's not a must for now, but I'd be curious to see more pydoc on the elements from the framework so that I can wrap my head around them more easily : )